ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -203,10 +216,19 @@
        context
        (ingest-events/provider-autocomplete-suggestion-reindexing-event provider)))))
 
+(defn bulk-granule-update-task-table-cleanup
+  "Clean up completed bulk granule update tasks that are older than the configured age"
+  [context]
+  (granule-bulk-update/cleanup-bulk-granule-tasks context))
+
 (def-stateful-job BulkUpdateStatusTableCleanup
   [_ system]
   (bulk-update-status-table-cleanup {:system system}))
 
+(def-stateful-job BulkGranUpdateTaskCleanup
+  [_ system]
+  (bulk-granule-update-task-table-cleanup {:system system}))
```

Comment:
Change this line to `(granule-bulk-update/cleanup-bulk-granule-tasks {:system system}))`